src/main/java/ua/danit/final_project/controllers/WorkShiftController.java

```diff
@@ -53,6 +53,7 @@ public WorkShiftController(WorkCommentService workCommentService,
   @PostMapping("/comment")
   public ShiftCommentDto createComment(@RequestBody ShiftCommentDto shiftCommentDto) throws JsonProcessingException {
     User userFromToken = getCurrentUser();
+    ShiftCommentDto ex = shiftCommentDto;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "ex" local variable. ([squid:S1481](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1481))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=167&issues=AWbwEn5XsSodgw_KsuuE&open=AWbwEn5XsSodgw_KsuuE)

[ISSUE_KEY]: AWbwEn5XsSodgw_KsuuE